src/stores/widgets/StopGapWidgetDriver.ts

```diff
@@ -366,4 +368,50 @@ export class StopGapWidgetDriver extends WidgetDriver {
             client.off(ClientEvent.TurnServersError, onTurnServersError);
         }
     }
+
+    public async readEventRelations(
+        eventId: string,
+        roomId?: string,
+        relationType?: string,
+        eventType?: string,
+        from?: string,
+        to?: string,
+        limit?: number,
+        direction?: 'f' | 'b',
+    ): Promise<IReadEventRelationsResult> {
+        const client = MatrixClientPeg.get();
+        const dir =
+            direction && Object.values(Direction as Record<string, string>).includes(direction)
+                ? direction as Direction
+                : undefined;
+        const room = roomId !== undefined ? roomId : RoomViewStore.instance.getRoomId();
```

Comment:
"room" typically means a `Room` object in our code - we should be able to redefine the `roomId` variable with `roomId = roomId ?? RoomViewStore.instance.getRoomId();`